zips/zip-0032.rst

```diff
@@ -439,7 +448,7 @@ Orchard child key derivation
 
 $\mathsf{CKDsk}((\mathsf{sk}_{par}, \mathsf{c}_{par}), i)$ $\rightarrow (\mathsf{sk}_{i}, \mathsf{c}_i)$ :
 
-- Return $\mathsf{CKDh}^\mathsf{Orchard}((\mathsf{sk}_{par}, \mathsf{c}_{par}), i)$
+- Return $\mathsf{CKDh}^\mathsf{Orchard}((\mathsf{sk}_{par}, \mathsf{c}_{par}), i, \texttt{“”})$
```

Comment:
Is the empty string the best way to express this? An alternative is just `[]` to represent the empty byte string.